VideoAPI/Video.API/Controllers/ConsultationController.cs

```diff
@@ -240,7 +245,77 @@ public async Task<IActionResult> StartPrivateConsultationWithEndpointAsync(Endpo
 
             return Accepted();
         }
-        
+
+        [HttpPost("start")]
+        [SwaggerOperation(OperationId = "StartPrivateConsultation")]
+        [ProducesResponseType((int) HttpStatusCode.Accepted)]
+        [ProducesResponseType((int) HttpStatusCode.NotFound)]
+        [ProducesResponseType((int) HttpStatusCode.Unauthorized)]
+        [ProducesResponseType((int) HttpStatusCode.BadRequest)]
+        public async Task<IActionResult> StartConsultationRequestAsync(StartConsultationRequest request)
+        {
+            Room room;
+            try
+            {
+                room = await GetAvailableConsultationRoom(request);
+            }
+            catch (ConferenceNotFoundException e)
```

Comment:
would this catch work?

Reply:
Moved the try catch inside the GetAvailableConsultationRoom method